examples/overwriteBCs.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+# This script demonstrates how to run and use the overwritebc option
+CGNS_INFILE="717_wl_L2.cgns"
+CGNS_OUTFILE="717_wl_L2_overwriteBCs.cgns"
+BC_FILE="overwriteBCs_bcFile"
+WRITE_FILE=true
+
+# Prepare the BC input file we want to use. Each line has the following format:
+# <block number> <faceID> <BCType> <family> [dataset]
+
+if [[ $WRITE_FILE -eq true ]]; then
+cat << 'EOF' > $BC_FILE
+1 kLow bcwallinviscid wall_invicid
```

Comment:
Very minor thing, the family name `wall_inviscid` has a typo here and in the following ` examples/overwriteBCs_bcFile` ;)